Gemfile.lock

```diff
@@ -347,7 +347,7 @@ GEM
       json (>= 1.8, < 3)
       simplecov-html (~> 0.10.0)
     simplecov-html (0.10.2)
-    sitemap_generator (5.3.1)
+    sitemap_generator (6.1.0)
```

Comment:
Have you verified that any breaking changes (if there are any), won't affect us?

Reply:
[There are backward incompatible changes](https://github.com/kjvarga/sitemap_generator/blob/master/CHANGES.md#600

Reply:
Below is the error I received when trying to "publish to production" before making the change in the GemFile
```
Completed 500 Internal Server Error in 27387ms (ActiveRecord: 971.9ms)


undefined method `yaml_as' for SitemapGenerator::BigDecimal:Class
Did you mean?  yaml_tag excluded from capture: DSN not set

NoMethodError (undefined method `yaml_as' for SitemapGenerator::BigDecimal:Class
Did you mean?  yaml_tag):

app/models/gibct_site_mapper.rb:32:in `generate_sitemap'
app/models/gibct_site_mapper.rb:11:in `initialize'
app/controllers/dashboards_controller.rb:63:in `new'
app/controllers/dashboards_controller.rb:63:in `push'

NameError: uninitialized constant Mime::HTML
	from /Users/matthewroth/.rbenv/versions/2.5.7/lib/ruby/gems/2.5.0/gems/web-console-2.3.0/lib/web_console/middleware.rb:58:in `acceptable_content_type?'
```

Reply:
So [this](https://github.com/kjvarga/sitemap_generator/blob/master/CHANGES.md#601) is the real reason this upgrade is needed.

Reply:
So do gem`'sitemap_generator', '6.0.0'` in GemFile ?

Reply:
Or close this PR and include gem upgrade in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/579 ?

Reply:
I think it's find to upgrade to the latest (6.1.0).  Bill just wants to make sure that since this is a "major" version upgrade that any breaking changes in the gem won't affect us.
I don't think it will; I don't recognize anything mentioned as being used in this repo (AWS SDK, S3, Fog & Wave adapters)

I was just pointing out why you are getting the error you reported.

And I do think it's good to take care of this _before_ the Ruby upgrade.